bitnami/postgresql/README.md

```diff
@@ -144,6 +144,9 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
 | `master.extraVolumeMounts`                    | Additional volume mounts to add to the pods (postgresql master)                                                                                                           | `[]`                                                          |
 | `master.extraVolumes`                         | Additional volumes to add to the pods (postgresql master)                                                                                                                 | `[]`                                                          |
 | `master.sidecars`                                         | Add additional containers to the pod                                                                                                                          | `[]`                                                     |
+| `master.service.type`                         | Kubernetes Master Service type (ovverides the service.type for differentiating from slave if desired)                                                                     | `nil`                                                         |
```

Comment:
I would change this by:
```
Allow to use a diferent service type for the master.             
```
And in the same way for the rest of comments

Reply:
Will do